app/views/accounts/_header.html.haml

```diff
@@ -1,6 +1,9 @@
 .public-account-header{:class => ("inactive" if account.moved?)}
   .public-account-header__image
-    = image_tag account.header.url, class: 'parallax'
+    - if current_account&.user&.setting_auto_play_gif
```

Comment:
Why using blocks here if you just need to switch the URL. Ternary operator should do

Reply:
For consistency with the code for avatars that is just a few lines below:
https://github.com/tootsuite/mastodon/pull/9515/files#diff-282ac2735bdc933e88272b729a2faf8bL6

Reply:
OK but that code is bad too. I think ideally it would be extracted to a helper method altogether.